redash/settings.py

```diff
@@ -73,6 +73,8 @@ def all_settings():
 CELERY_BROKER = os.environ.get("REDASH_CELERY_BROKER", REDIS_URL)
 CELERY_BACKEND = os.environ.get("REDASH_CELERY_BACKEND", CELERY_BROKER)
 CELERY_TASK_RESULT_EXPIRES = int(os.environ.get('REDASH_CELERY_TASK_RESULT_EXPIRES', 3600))
+CELERYD_PREFETCH_MULTIPLIER = int(os.environ.get('REDASH_CELERYD_PREFETCH_MULTIPLIER', 4))
+CELERY_ACKS_LATE = parse_boolean(os.environ.get('REDASH_CELERY_ACKS_LATE', "false"))
```

Comment:
Not just too clever, but my experience at work has been that Celery is occasionally buggy when you start flipping more advanced options like this. The code is simply less well-tested and fewer folks have incentive to maintain these parts of the celery code. I like Celery for many things, but I agree with @arikfr that it's best to try and minimize usage of the non-standard options unless forced otherwise (which does happen from time to time).